bin/runtime/src/lib.rs

```diff
@@ -825,18 +826,18 @@ impl InstanceFilter<RuntimeCall> for ProxyType {
                         | RuntimeCall::NominationPools(..)
                 )
             }
+            ProxyType::Nomination => {
+                matches!(
+                    c,
+                    RuntimeCall::NominationPools(pallet_nomination_pools::Call::nominate { .. })
```

Comment:
At this point it doesn't seem like we need `NominationPools` here. Also I'm not sure what the `nominate` call does exactly, so I would recommend dropping it.